src/main/scala/nl/knaw/dans/easy/multideposit/CommandLineOptions.scala

```diff
@@ -43,7 +43,7 @@ object CommandLineOptions extends DebugEnhancedLogging {
       outputDepositDir = opts.outputDepositDir().getAbsoluteFile,
       datamanager = opts.datamanager(),
       depositPermissions = DepositPermissions(props.getString("deposit.permissions.access"), props.getString("deposit.permissions.group")),
-      formatsFile = new File(props.getString("formats.file")),
+      formatsFile = new File(homeDir, "cfg/formats.txt"),
```

Comment:
`formats.txt` is a configuration file. The application may therefore assume that it is the config directory.